src/pages/index.astro

```diff
@@ -25,8 +25,14 @@ const { data: projectData } = await storyblokApi.get('cdn/stories', {
 
 const projects = projectData.stories;
 const story = homeData.story;
+
+// Extract text from the aboutText content
+const aboutTextString = story.content.aboutText.content
+  .map(paragraph => paragraph.content
+  .map(item => item.text)
+  .join(''));
```

Comment:
You might be able to use flat map here so that there is less nesting.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

It's a subtle change, but more "flat"

```
const aboutTextString = story.content.aboutText.content
  .flatMap(paragraph => paragraph.content)
  .map(item => item.text)
  .join('');
```